documentation.go

```diff
@@ -138,16 +244,13 @@ func (c *documentationCommand) formatCommand(ref commandReference) string {
 		formatted += "\n"
 	}
 
-	// See Also
-	if len(ref.command.Info().SeeAlso) > 0 {
-		formatted += "## See Also\n"
-		for _, s := range ref.command.Info().SeeAlso {
-			formatted += fmt.Sprintf("[%s](#%s)\n", s, s)
-		}
-		formatted += "\n"
+	// Options
+	formattedFlags := c.formatFlags(ref.command)
+	if len(formattedFlags) > 0 {
+		formatted += "## Options\n" + formattedFlags + "\n"
 	}
 
-	formatted += "---\n"
+	formatted += "---\n\n"
```

Comment:
why do we need this double newline?

Reply:
In best practices ([here](https://www.markdownguide.org/basic-syntax#horizontal-rule-best-practices)), horizontal requires an empty line before other headings